README.md

```diff
@@ -7,32 +7,37 @@ A GitHub Action that will run [Flyway](https://flywaydb.org/) against a specifie
+| `baseline-version`            | false       | 0       | The baseline version to send to the flyway command.                                                                                                                                                                                        |
+| `managed-schemas`             | true        | N/A     | A comma separated list of schemas that are to be managed by Flyway.MigrationHistory.                                                                                                                                                       |
+| `enable-out-of-order`         | false       | false   | A switch that allows new migrations that are a lower version number than a current migration to be run.                                                                                                                                    |
+| `validate-migrations`         | false       | true    | A switch determining whether flyway should validate the migration scripts before running them.                                                                                                                                             |
+| `use-integrated-security`     | false       | false   | A switch defining whether or not to use integrated security. If not provided, a password should be.                                                                                                                                        |
+| `use-azure-managed-identity`  | false       | false   | A switch that can be used to indicate that an Azure Managed Identity should be used to authenticate with the SQL Server.                                                                                                                   |
+| `use-azure-service-principal` | false       | false   | A switch that can be used to indicate that an Azure Active Directory Service Principal will be used to authentication with the SQL Server.                                                                                                 |
+| `azure-msi-client-id`         | false       | N/A     | The Azure Client Id of the Managed Identity used to login to the database. Must be specified if the Managed Identity is a User-Assigned Managed Identity and the use-azure-managed-identity flag is set.                                   |
+| `username`                    | true        | N/A     | The username of the user making the changes, which is put into the MigrationHistory table, and also to login with if not using integrated security. This should be the Service Principal ID if use-azure-service-principal is set to true. |
+| `password`                    | false       | N/A     | The password for the user making changes if not using integrated security. This should be the Service Principal Secret if use-azure-service-principal is set to true.                                                                      |
```

Comment:
For the description for both of these, would you use it with any other scenario or just the use-azure-service-principal like the user/password authentication?

Reply:
Yep only those two scenarios, user/password or service principal auth. Is there wording I could use to make that more clear?